_automating-configurations/api/get-workflow-steps.md

```diff
@@ -7,10 +7,7 @@ nav_order: 50
 
 # Get workflow steps
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/flow-framework/issues/475).    
-{: .warning}
-
-OpenSearch validates workflows by using the validation template that lists the required inputs, generated outputs, and required plugins for all steps. For example, for the `register_remote_model` step, the validation template appears as follows:
+This API returns a list of workflow steps, including their required inputs, outputs, default timeout value, and required plugins  For example, for the `register_remote_model` step, the Get Workflow Steps API returns the following information:
```

Comment:
Should "timeout value" be "timeout values" (plural)?

Reply:
@owaiskazi19 I think it's fine to keep it plural since "steps" is plural. So there are timeouts for a number of steps (even though each has only one timeout), if that makes sense.